README.md

```diff
@@ -1,707 +1,1799 @@
 # Reselect
 
+[![npm package][npm-badge]][npm][![Coveralls][coveralls-badge]][coveralls][![GitHub Workflow Status][build-badge]][build]![TypeScript][typescript-badge]
+
 A library for creating memoized "selector" functions. Commonly used with Redux, but usable with any plain JS immutable data as well.
 
-- Selectors can compute derived data, allowing Redux to store the minimal possible state.
-- Selectors are efficient. A selector is not recomputed unless one of its arguments changes.
-- Selectors are composable. They can be used as input to other selectors.
+- Selectors can compute derived data, allowing `Redux` to store the minimal possible state.
+- Selectors are efficient. A selector is not recomputed unless one of its arguments changes.
+- Selectors are composable. They can be used as input to other selectors.
+
+The **Redux docs usage page on [Deriving Data with Selectors](https://redux.js.org/usage/deriving-data-selectors)** covers the purpose and motivation for selectors, why memoized selectors are useful, typical `Reselect` usage patterns, and using selectors with `React-Redux`.
+
+## Installation
+
+### Redux Toolkit
+
+While `Reselect` is not exclusive to `Redux`, it is already included by default in [the official Redux Toolkit package](https://redux-toolkit.js.org) - no further installation needed.
+
+```ts
+import { createSelector } from '@reduxjs/toolkit'
+```
+
+### Standalone
+
+For standalone usage, install the `Reselect` package:
+
+#### Using `npm`
```

Comment:
:hand: Two notes:

- Not sure we to list all four package managers separately
- However many we leave in, consolidate them into a single code block ( `bash` language, and a `# NAME` comment above, like:

```bash
# NPM
npm install reselect

# Yarn
yarn add reselect
```

Reply:
I only did that to make copy pasting with a single click easier. Do you still want me to change it?

Reply:
I don't think we need to optimize for people copy-pasting the instructions to install one package, yeah :)  We also use the code block approach in the other Redux docs sites for this step anyway.

Reply:
ok :(